config/locales/i18n/en.yml

```diff
@@ -6,3 +6,4 @@ en:
       en: English
       es: Español
       fr: Français
+      zh: Chinese
```

Comment:
Re: Previous comment, I was thinking of language picker, which is here. The other comment is about email language preference, but same note applies.

Reply:
Yes, but I don't think we have that translated yet.

Reply:
Circling back here. Chinese needs to be changed to Chinese (Simplified). We should stay consistent with other areas of the IdP and change so that each language is displayed in the language it is describing.